tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astroquery.sdss import SDSS\n",
+      "sdss = SDSS.query_region(coordinates=hcg7_center, radius=20*u.arcmin, spectro=True, photoobj_fields=['ra','dec','u','g','r','i','z'])"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "`astroquery` gives us back `astropy.table.Table` objects, which we can write to disk so that we don't have to run the query every time we want to use it."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "sdss.write('HCG7_SDSS_photo.dat', format='ascii')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now we simply load them back from disk. You can skip this step as long as your astroquery call suceeded, as the tables will be identical given that you just wrote them."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "sdss = Table.read('HCG7_SDSS_photo.dat', format='ascii')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Ok, so we have a catalog of objects we got from the SDSS.  Now lets say you have your own catalog of objects in the same field that you want to match to this SDSS catalog.  In this case, we will use a catalog extracted from the [2MASS](http://www.ipac.caltech.edu/2mass/).  We first load up this catalog into python."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "twomass = Table.read('HCG7_2MASS.tbl', format='ascii')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now to do matching we need `SkyCoord` objects.  We'll have to build these from the tables we loaded, but it turns out that's pretty straightforward: we just grab the RA and dec columns from the table and provide them to the `SkyCoord` constructor."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "sdss # just to see an example of the format"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "twomass # just to see an example of the format"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "coo_sdss = SkyCoord(sdss['ra']*u.deg, sdss['dec']*u.deg)\n",
+      "coo_twomass = SkyCoord(twomass['ra'], twomass['dec'])"
```

Comment:
I could see getting confused seeing this for the first time--nowhere has it been mentioned that `SkyCoord` can be made with a whole array of coordinates.

Also (actual question *I* have): Who do we have to manually attach the units to the 'ra' and 'dec' columns when it appears that the table already has units for those columns?  Should this have used a QTable, maybe?